content/100-getting-started/01-quickstart.mdx

```diff
@@ -134,12 +134,12 @@ If you're interested in learning more about Prisma, the [Concepts](../../concept
 
 If you are looking for guidance and workflows the [Guides](../../guides) section has a plethora of content that covers subjects such as [Database guides](../../guides/database), [Testing](../../guides/testing), [Deployment](../../guides/deployment) and [Optimizations](../../guides/performance-and-optimization).
 
-Alternatively, if you want to include Prisma in an **existing project** the [Add to existing project](./setup-prisma/add-to-existing-project-typescript-postgres) guide explains some core concepts as it guides you through intergrating Prisma into your workflow.
+Alternatively, if you want to include Prisma in an **existing project** the [Add to existing project](./setup-prisma/add-to-existing-project/relational-databases-typescript-postgres) guide explains some core concepts as it guides you through integrating Prisma into your workflow.
 
-Or perhaps you're looking at **starting a new project with Prisma**? The [Start from scratch](./setup-prisma/start-from-scratch-typescript-postgres) section guides you through creating a project, connecting your database and querying your data from scratch.
+Or perhaps you're looking at **starting a new project with Prisma**? The [Start from scratch](./setup-prisma/start-from-scratch/relational-databases-typescript-postgres) section guides you through creating a project, connecting your database and querying your data from scratch.
```

Comment:
This is my part. Going to pull this in and take a look again. Will report back

Reply:
Ok i just looked again. The reason it was decided to split it out was that we cant conditionally show/not show pages in the sidenav. Although mongo has mostly the same setup, it doesnt have introspection or migration yet so there would be a ghost link in the sidenav. 

By giving it its own section we get rid of that issue. Another reason was the google foo

Reply:
We could maybe expand on the actual getting started page to link to either the relational db or mongo guides. Right now they default to the relational and dont even mention the mongo guide. What do you think about that @mhwelander ?

Reply:
FWIW, I think it's fine being a separate page or section. Probably easiest to explain in code. 

Right now the toggle looks something like this:

```html
<ul>
  <li>
    <a href="#mysql">MySQL</a>
  </li>
  <li>
    <a href="#postgres">Postgres</a>
  </li>
</ul>
```

I'd propose we do this instead:

```diff
<ul>
  <li>
    <a href="#mysql">MySQL</a>
  </li>
  <li>
    <a href="#postgres">Postgres</a>
  </li>
+  <li>
+    <a href="/mongodb">MongoDB</a>
+  </li>
</ul>
```

For both pages.

Reply:
That, unfortunately doesnt make much sense if you are on the relational database page. The toggle is shown on the mongo pages because it has to be there for the nav buttons at the bottom. I dont understand exactly why that is but i originally didnt have the mongo toggle on the mongo pages as its just mongo 😅

If the user navigates to the getting started page we can give them some options to go to the relational database guide or mongo. 

But to add mongo to the toggle on the relational database page will, in my opinion, confuse the user and our message